### ISSUE TITLE
[Fedora 30] No module named 'cdio'

### ISSUE THREAD

**Issue by remjg** [whipper-team/whipper#471]
I have just installed Whipper on Fedora 30 from the repositories, but it throws the following error at launch: 

      File "/usr/bin/whipper", line 11, in <module>
        load_entry_point('whipper==0.9.0', 'console_scripts', 'whipper')()
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
        return get_distribution(dist).load_entry_point(group, name)
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2793, in load_entry_point
        return ep.load()
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2411, in load
        return self.resolve()
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2417, in resolve
        module = __import__(self.module_name, fromlist=['__name__'], level=0)
      File "/usr/lib64/python3.7/site-packages/whipper/command/main.py", line 11, in <module>
        from whipper.command import cd, offset, drive, image, accurip, mblookup
      File "/usr/lib64/python3.7/site-packages/whipper/command/cd.py", line 22, in <module>
        import cdio
    ModuleNotFoundError: No module named 'cdio'

And yet, the following packages are installed: 

    libcdio.x86_64        2.0.0-4.fc30         @fspin-release-2019-08-16-1565957524
    pycdio.x86_64         2.0.0-4.fc30         @fedora                             
    whipper.x86_64        0.9.0-1.fc30         @updates 

As a workaround, I tried to install `pycdio` using the command line `pip3 install --user pycdio` but it failed (see the attached [pip3-install-pycdio.log](https://github.com/whipper-team/whipper/files/4293324/pip3-install-pycdio.log) for details).

The line bugs me in particular (since I already have le 2.0.0-4.fc30 version): 

    AssertionError: Need at least libcdio 2.0.0 to use this

Maybe it's working on Fedora 31, but I can't upgrade my system for now. Any clue ?

**Comment by remjg** [comment]
The `pycdio` package provided in Fedora 30 doesn't seem to have any python3.7 related files:

    $ dnf repoquery -l pycdio
    /usr/lib/.build-id
    /usr/lib/.build-id/28
    /usr/lib/.build-id/28/765a7b5a0265c7fde1d5b4d63dcb67766bc2bd
    /usr/lib/.build-id/cb
    /usr/lib/.build-id/cb/e0d5049ffbb3516c3f9ce0b97b3da55f9fbae5
    /usr/lib64/python2.7/site-packages/_pycdio.so
    /usr/lib64/python2.7/site-packages/_pyiso9660.so
    /usr/lib64/python2.7/site-packages/cdio.py
    /usr/lib64/python2.7/site-packages/cdio.pyc
    /usr/lib64/python2.7/site-packages/cdio.pyo
    /usr/lib64/python2.7/site-packages/iso9660.py
    /usr/lib64/python2.7/site-packages/iso9660.pyc
    /usr/lib64/python2.7/site-packages/iso9660.pyo
    /usr/lib64/python2.7/site-packages/pycdio-2.0.0-py2.7.egg-info
    /usr/lib64/python2.7/site-packages/pycdio-2.0.0-py2.7.egg-info/PKG-INFO
    /usr/lib64/python2.7/site-packages/pycdio-2.0.0-py2.7.egg-info/SOURCES.txt
    /usr/lib64/python2.7/site-packages/pycdio-2.0.0-py2.7.egg-info/dependency_links.txt
    /usr/lib64/python2.7/site-packages/pycdio-2.0.0-py2.7.egg-info/top_level.txt
    /usr/lib64/python2.7/site-packages/pycdio.py
    /usr/lib64/python2.7/site-packages/pycdio.pyc
    /usr/lib64/python2.7/site-packages/pycdio.pyo
    /usr/lib64/python2.7/site-packages/pyiso9660.py
    /usr/lib64/python2.7/site-packages/pyiso9660.pyc
    /usr/lib64/python2.7/site-packages/pyiso9660.pyo
    /usr/share/doc/pycdio
    /usr/share/doc/pycdio/AUTHORS
    /usr/share/doc/pycdio/ChangeLog
    /usr/share/doc/pycdio/NEWS
    /usr/share/doc/pycdio/README.rst
    /usr/share/doc/pycdio/THANKS
    /usr/share/licenses/pycdio
    /usr/share/licenses/pycdio/COPYING

According to this [Fedora bug report](https://bugzilla.redhat.com/show_bug.cgi?id=1775536), there has been some issue with the version of `pycdio` packaged in Fedora.

I installed manually a new package called `python3-pycdio` from Fedora 31 that I found [here](https://fedora.pkgs.org/31/fedora-updates-x86_64/python3-pycdio-2.0.0-7.fc31.x86_64.rpm.html).

My probleme seems to be solved... Hopefully, someone on the Fedora 31 (the latest at this time) should be okay using Whipper.

**Comment by JoeLametta** [comment]
> My probleme seems to be solved... Hopefully, someone on the Fedora 31 (the latest at this time) should be okay using Whipper.

Thanks for explaining how you solved the issue.
Pinging @mruszczyk

**Comment by mruszczyk** [comment]
Thank you for the bug report. I'll spin up a fedora 30 box tomorrow, it looks like the original pycdio package got orphaned and future work did get moved to python-pycdio 4 months ago. I'll get this sorted in the main repos.

**Comment by mruszczyk** [comment]
Strange, on my system I just checked attempting to install pycdio does in fact install python3-pycdio. Unsure why that would be different for you, and python3-pycdio should be available in the f30 official repositories. 

Either way I've submitted a new build to testing that explicitly requires the python3 version of pycdio. It should land in f30 updates repo in about a week if you don't want to install the testing rpm. 

https://bodhi.fedoraproject.org/updates/FEDORA-2020-9d88e77dc3
